README.md

```diff
@@ -17,17 +17,24 @@ Header-only C++20 library that provides containers with the following properties
 
 # Features
 
-* `FixedVector` - Vector implementation with `std::vector` API and "fixed container" properties
-* `FixedMap`/`FixedSet` - Red-Black Tree map/set implementation with `std::map`/`std::set` API and "fixed container" properties.
-* `EnumMap`/`EnumSet` - For enum keys only, Map/Set implementation with `std::map`/`std::set` API and "fixed container" properties. O(1) lookups.
-* `FixedDeque` - Deque implementation with `std::deque` API and "fixed container" properties
-* `FixedQueue` - Queue implementation with `std::queue` API and "fixed container" properties
-* `FixedStack` - Stack implementation with `std::stack` API and "fixed container" properties
-* `FixedCircularDeque` - Circular buffer implementation with `std::deque` API and "fixed container" properties
-* `FixedCircularQueue` - Circular buffer implementation with `std::queue` API and "fixed container" properties
-* `FixedString` - String implementation with `std::string` API and "fixed container" properties
+* The following are equivalent in functionality/API with std-containers and have "fixed container properties":
+
+   | fixed-container      | std-container equivalent                        |
+   |:---------------------|:------------------------------------------------|
+   | `FixedVector`        | `std::vector`                                   |
+   | `FixedDeque`         | `std::deque`                                    |
+   | `FixedQueue`         | `std::queue`                                    |
+   | `FixedStack`         | `std::stack`                                    |
+   | `FixedCircularDeque` | `std::deque` API with Circular Buffer semantics |
+   | `FixedCircularQueue` | `std::queue` API with Circular Buffer semantics |
+   | `FixedString`        | `std::string`                                   |
+   | `FixedMap`           | `std::map`                                      |
+   | `FixedSet`           | `std::set`                                      |
+   | `EnumMap`            | `std::map` for enum keys only                   |
+   | `EnumSet`            | `std::set` for enum keys only                   |
+   | `EnumArray`          | `std::array` but with typed accessors           |
```

Comment:
This doesn't really explain the difference between an `EnumMap` and an `EnumArray`, they have very similar interfaces when you're talking about the key/index type being a finite set of values.